<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AntStatusBoard</class>
 <widget class="QDialog" name="AntStatusBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridlayoutmain">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="basicTag">
      <attribute name="title">
       <string>setting</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>107</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayoutwp" stretch="3,7">
         <item>
          <widget class="QLabel" name="workspaceLabel">
           <property name="text">
            <string>work space</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="workspaceEdit"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayoutos" stretch="3,7">
         <item>
          <widget class="QLabel" name="outspaceLabel">
           <property name="text">
            <string>out space</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="outspaceEdit"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayoutlf" stretch="3,7">
         <item>
          <widget class="QLabel" name="logfileLabel">
           <property name="text">
            <string>log file</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="logfileEdit"/>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayoutcs" stretch="3,7">
         <item>
          <widget class="QLabel" name="cropsizeLabel">
           <property name="text">
            <string>crop size</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cropsizeEdit">
           <property name="placeholderText">
            <string>128x128</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayoutzf" stretch="3,7">
         <item>
          <widget class="QLabel" name="zoomfLabel">
           <property name="text">
            <string>zoom factor</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="zoomfBox"/>
         </item>
        </layout>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>106</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cancleButton">
           <property name="text">
            <string>Cancle</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="docTag">
      <attribute name="title">
       <string>help</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTextBrowser" name="docBrowser">
         <property name="source">
          <url>
           <string>qrc:/doc/ant-help.html</string>
          </url>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancleButton</sender>
   <signal>clicked()</signal>
   <receiver>AntStatusBoard</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
